### PR TITLE
Added SSH version as a policy control. Fixes #154

### LINKED ISSUE #154: Add SSH version as a policy control
Perhaps we want to require that SSH be version 2.0

Maybe this would be a bad example?

 "ssh_version": 1.99,

### PR DESCRIPTION
**Pull request by yashmehrotra**
We just need to add `ssh_version: <minimum>` to the `yml` file.

### CODE REVIEW

**Comment by claudijd** [comment]
@yashmehrotra please feel free to add 2.0 to modern and intermediate policies here in this PR => https://github.com/mozilla/ssh_scan/tree/master/policies

**Comment by claudijd** [comment]
@yashmehrotra also, feel free to join us in Glitter chat if you'd like.  I'd invite you directly, but don't have your email (https://gitter.im/mozilla-ssh_scan/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

File: lib/ssh_scan/policy_manager.rb [hunk 1]
```diff
@@ -104,6 +104,16 @@ def out_of_policy_auth_methods
...
 
+    def minimum_ssh_version
+      target_ssh_version = @result[:ssh_version]
+      if @policy.ssh_version
+        if target_ssh_version < @policy.ssh_version
+          return @policy.ssh_version
+        end
+      end
+      return false
+    end
```
Comment (claudijd) [inline lib/ssh_scan/policy_manager.rb]: Returning false or string is a bit weird here.

Maybe we need to improve terminology here.

Maybe there is a method that returns true/false, one that returns the version we saw, and one that returns the version we want?  I feel like that might be a little more deliberate.

File: spec/ssh_scan/policy_manager_spec.rb [hunk 2]
```diff
@@ -192,4 +192,25 @@
...
+                  "auth_methods:\n- publickey\n" +
+                  "ssh_version: 2.0\n"
+    result = {
+                :ssh_version => 1.9
+             }
+
+    it "should load all the attributes properly" do
+      policy = SSHScan::Policy.from_string(yaml_string)
+      policy_manager = SSHScan::PolicyManager.new(result, policy)
+      expect(policy_manager.minimum_ssh_version()).to eql(2.0)
```
Comment (claudijd) [inline spec/ssh_scan/policy_manager_spec.rb]: Same vein as the method criticism, I think we need more deliberate methods here.  We're using one method for multiple uses and I don't think it's direct enough.

Comment (yashmehrotra) [inline spec/ssh_scan/policy_manager_spec.rb]: @claudijd So what do you suggest for this ?

Comment (claudijd) [inline spec/ssh_scan/policy_manager_spec.rb]: very the response of "policy_mgr.compliance_results" that you expect

**Comment by yashmehrotra** [comment]
@claudijd Hi, I used a new approach, where the function that checks the minimum ssh version only returns a boolean. I hope its good.

**Comment by claudijd** [comment]
@yashmehrotra thanks!

**Pull request merged**